src/components/CommandBar/CommandBarBasicInput.tsx

```diff
@@ -70,6 +70,11 @@ function CommandBarBasicInput({
         <span className="capitalize px-2 py-1 rounded-l bg-chalkboard-100 dark:bg-chalkboard-80 text-chalkboard-10 border-b border-b-chalkboard-100 dark:border-b-chalkboard-80">
           {arg.displayName || arg.name}
         </span>
+        {arg.inputType === 'tagDeclarator' && (
+          <span className="pl-2 py-1 -mr-2 border-b border-b-chalkboard-100 dark:border-b-chalkboard-80">
+            $
+          </span>
+        )}
```

Comment:
Figured we could use a little extra UI thing here. Could become its own Input type later, seems enough for now tho?

Reply:
Yes I think users that are both aware of how to use `tagStart` and `tagEnd` will also be users who might accidentally include the `$` in the form field.